tests/files/axis/availability_with_loan_and_hold.xml

```diff
@@ -1 +1 @@
Date><availability><availableFormats><formatName>ePub</formatName><formatName>Blio</formatName></availableFormats><availableCopies>0</availableCopies><totalCopies>2</totalCopies><Checkouts/><Holds/><holdsQueueSize>1</holdsQueueSize><holdsQueuePosition>1</holdsQueuePosition><isInHoldQueue>true</isInHoldQueue><isReserved>false</isReserved><isCheckedout>false</isCheckedout><updateDate>7/22/2015 4:05:11 PM</updateDate></availability><minLoanPeriod>3</minLoanPeriod><maxLoanPeriod>21</maxLoanPeriod><defaultLoanPeriod>21</defaultLoanPeriod><RTV>true</RTV></title><title xsi:type="Axis360AvailCirculation"><titleId>0015176429</titleId><productTitle>Safe at Last</productTitle><titleUrl>http://nypl.axis360.baker-taylor.com/Title?itemId=0015176429</titleUrl><contributor>Banks, Maya </contributor><isbn>9780062312518</isbn><subject>FICTION / Romance / Contemporary; FICTION / Romance / Suspense</subject><series>Slow Burn</series><publisher>HarperCollins</publisher><language>ENGLISH</language><audience>General Adult</audience><imprint>Avon Books</imprint><annotation>When Tori Devereaux begins experiencing terrifying visions that start to come true, Dane Elliot resolves to protect her from a particularly harrowing prediction that threatens to claim her forever. By a #1 New York Times best-selling author. Original. 200,000 first printing.</annotation><narrator/><edition/><runtime/><fileSize/><addedDate>6/25/2015 10:15:01 PM</addedDate><publicationDate>6/23/2015</publicationDate><availability><availableFormats><formatName>ePub</formatName><formatName>Blio</formatName></availableFormats><availableCopies>29</availableCopies><totalCopies>30</totalCopies><Checkouts/><Holds/><holdsQueueSize>0</holdsQueueSize><isInHoldQueue>false</isInHoldQueue><isReserved>false</isReserved><isCheckedout>true</isCheckedout><checkoutFormat>ePub</checkoutFormat><checkoutStartDate>7/22/2015 5:40:27 PM +00:00</checkoutStartDate><checkoutEndDate>8/12/2015 5:40:27 PM +00:00</checkoutEndDate><downloadUrl>http://fulfillment/</downloadUrl><updateDate>7/8/2015 6:16:45 AM</updateDate></availability><minLoanPeriod>3</minLoanPeriod><maxLoanPeriod>21</maxLoanPeriod><defaultLoanPeriod>21</defaultLoanPeriod><RTV>true</RTV></title><title xsi:type="Axis360AvailCirculation"><titleId>1111111111</titleId><productTitle>Test</productTitle><isbn>12345</isbn><language>FRENCH</language><availability><availableCopies>0</availableCopies><totalCopies>2</totalCopies><Checkouts/><Holds/><holdsQueueSize>1</holdsQueueSize><holdsQueuePosition>1</holdsQueuePosition><isInHoldQueue>false</isInHoldQueue><isReserved>true</isReserved><reservedEndDate>1/1/2015 1:11:11 PM +00:00</reservedEndDate><isCheckedout>false</isCheckedout><updateDate>7/22/2015 4:05:11 PM</updateDate></availability><minLoanPeriod>3</minLoanPeriod><maxLoanPeriod>21</maxLoanPeriod><defaultLoanPeriod>21</defaultLoanPeriod></title><status><code>0000</code><statusMessage>Availability Data is Successfully retrieved.</statusMessage></status></availabilityResponse>
```

Comment:
This is another out of date or contrived example. The availability information for titleId 1111111111 was missing. This made the file fail validation. Looking at real API responses and the docs, this information should not be missing. I so I modified the example to add in the required availability info.